src/visualizers/visualizerSpace.js

```diff
@@ -120,8 +120,8 @@ const SpaceVisualizer = function () {
     return [offsetX, offsetY];
   };
   Polygon.prototype.drawPolygon = function () {
```

Comment:
This really doesn't seem idiomatic. Now that we have `class` support, we should NEVER see the `prototype` keyword and use prototype inheritance like this explicitly.

Reply:
It is probably because this is still using a function as an object. This need to be converted tho.

Reply:
I agree, that would seem like an ideal approach.

Reply:
You could probably extract a `Polygon` class for this.